glib/src/param_spec.rs

```diff
@@ -1225,15 +1236,15 @@ mod tests {
     fn test_param_spec_string() {
         let pspec = ParamSpecString::new(
             "name",
-            "nick",
-            "blurb",
+            None,
+            Some("blurb"),
```

Comment:
why you dropped the nick & not the blurb?

Reply:
Arbitrary choice as it's in tests. I can set all to None or Some if you wish.

Reply:
Right, never mind then.